app/admin/dashboard/article/page.tsx

```diff
@@ -1,90 +1,122 @@
-'use client';
+// 'use client';
```

Comment:
Please delete the comment if the changes are ready to signoff, we could use git blame to dissect the previous code base